src/components/organisms/AuthenticationModal/RegisterForm/RegisterForm.tsx

```diff
@@ -206,7 +228,7 @@ const RegisterForm: React.FunctionComponent<PropsType> = ({
           )}
         </div>
 
-        {venue.requiresTicketCode && (
+        {venue.access?.includes(VenueAccessMode.Codes) && (
```

Comment:
We seem to check this type of thing in a few places. Can we extract them all into a couple of `const`s.. eg:

```typescript
const hasVenueAccessEmails = venue.access?.includes(VenueAccessMode.Emails) ?? false;
const hasVenueAccessCodes = venue.access?.includes(VenueAccessMode.Codes) ?? false;
// etc
```